Source/ASDisplayNode+Yoga.mm

```diff
@@ -31,8 +32,17 @@ - (ASSizeRange)_locked_constrainedSizeForLayoutPass;
 
 @implementation ASDisplayNode (Yoga)
 
+- (ASDisplayNode *)yogaRoot {
+  ASDisplayNode *yogaRoot = self;
+  while(yogaRoot.yogaParent) {
+    yogaRoot = yogaRoot.yogaParent;
```

Comment:
Can we avoid calling `yogaParent` twice per node?

Reply:
Yes, also a space after "while ("